packages/forklift-console-plugin/src/components/common/ProjectNameSelect.tsx

```diff
@@ -27,6 +27,15 @@ export const ProjectNameSelect: React.FC<ProjectNameSelectProps> = ({
 }) => {
   const { t } = useForkliftTranslation();
 
+  // Fallback to "default" when the initial value does not exist within options
+  React.useEffect(() => {
+    const hasOptionValue = options.find((option) => option.name === value);
+
+    if (!hasOptionValue) {
+      onSelect(Namespace.Default);
```

Comment:
non admin users usually dont have access to Namespace.default

Reply:
Oh I see. I could just remove this effect then if that would make more sense.

Reply:
Removed this hook altogether. Let me know if there's any issue with that.